### PR TITLE
Czech translation for version 9.0

### PR DESCRIPTION
**Pull request by xmorave2**
Here is Czech translation for upcoming version 9.0.

I just didn't translate DDC23 domain as DDC is almost not used in Czech Republic - we usually use UDC in our libraries.

### CODE REVIEW

**Comment by xmorave2** [comment]
@black23 Could you please review it?

**Comment by demiankatz** [comment]
Thanks, @xmorave2! I'll wait for @black23's feedback before merging this. I agree that it's fine to skip the DDC23 domain if you don't need/use it. Regarding UDC, I opened #2488 to begin collecting UDC translations and to keep track of work needed to better support UDC natively. If you're interested in moving that forward at any time, let me know and I'd be happy to collaborate (but no pressure if it's not useful for you).

**Comment by xmorave2** [comment]
@demiankatz Thanks Demian, I am aware of #2488, but there are some licensing issues with official Czech UDC translations as far as I know. We are trying to investigate it more deeply and get the translation into VuFind if it would be possible.

**Comment by demiankatz** [comment]
Just following up to see if @black23 has had a chance to look at this. I imagine that @welblaud might also be willing to offer feedback if you're looking for another opinion.

**Comment by black23** [comment]
Hi Demian,
please give two days for review.

Thanks

Michal

st 22. 3. 2023 v 17:51 odesílatel Demian Katz ***@***.***>
napsal:

**Comment by demiankatz** [comment]
@black23, certainly, we still have plenty of time. :-)

File: languages/CookieConsent/cs.ini
Comment (janhejzl) [inline languages/CookieConsent/cs.ini]: změnilY

File: languages/cs.ini
Comment (janhejzl) [inline languages/cs.ini]: maybe 'Vašim datům' (uppercase V, just for consistency)?

File: languages/cs.ini
Comment (janhejzl) [inline languages/cs.ini]: dtto maybe 'Vašeho'

**Review by welblaud — APPROVED** [review]

File: languages/CookieConsent/cs.ini
Comment (xmorave2) [reply languages/CookieConsent/cs.ini]: Thanks, fixed

**Comment by xmorave2** [comment]
Thanks @welblaud for review. I fixed issues you found and also updated all translation with 'Váš', 'Vašeho', etc. to be uppercased for consistancy.

File: languages/CookieConsent/cs.ini
Comment (black23) [inline languages/CookieConsent/cs.ini]: "<p>Analytické cookies umožňují sledovat souhrnné informace o&nbsp;chování návštěvníků stránel.<br>Díky těmto informacím jsme schopni lépe přizpůsobit stránky uživatelům webu.</p><p>Jsou povoleny pouze na&nbsp;základě vašeho souhlasu.</p>"

File: languages/CookieConsent/cs.ini
Comment (black23) [inline languages/CookieConsent/cs.ini]: "<p>Funkční cookies jsou zapotřebí k zajištění základních funkcí webu.</p><p>Vzhledem k jejich významu a účelu jsou povoleny vždy a souhlas s jejich použitím není vyžadován.</p>"

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini:67]: "Zeptat se pokaždé"

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini]: Buďte první, kdo vytvoří štítek k temuto záznamu"

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini]: Na vaši e-mailovou adresu byly zaslány instrukce pro ověření e-mailové adresy. Vaši e-mailovou adresu nebudeme moci použít, dokud její ověření neprovedete."

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini]: "Podobné jednotky"

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini:1025]: "Profil byl upraven podle vašeho požadavku"

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini:606]: "Na tento dokument již máte rezervaci."

File: languages/cs.ini
Comment (black23) [inline languages/cs.ini]: "Knihovna neumožňuje vytvářet požadavky na rezervaci dokumentů, které nejsou vypůjčené."

**Review by black23 — COMMENTED** [review]
Some reviews.

File: languages/CookieConsent/cs.ini
Comment (xmorave2) [reply languages/CookieConsent/cs.ini]: Fixed. But just keep "Vašeho" instead "vašeho" you proposed - for consistency.

File: languages/CookieConsent/cs.ini
Comment (xmorave2) [reply languages/CookieConsent/cs.ini]: Fixed

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini:67]: It is in profile when you are selecting your preferred library. In that context I think my translation is slightly better.  Because accented word should be "pokaždé" not "zeptat",

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini]: Fixed

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini]: Fixed

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini]: Fixed

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini:1025]: We are always using "Vašeho" instead of "vašeho". So I am keeping it here too for consistency

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini:606]: I am keeping it as it is - the meaning is the same. But I think it is better to use "action" verb instead of  form with auxiliary verb.

File: languages/cs.ini
Comment (xmorave2) [reply languages/cs.ini]: Fixed

**Comment by xmorave2** [comment]
Thanks @black23 for review. I changed most of strings you proposed. There are some strings I decided to keep as they are. See reasons in my comments.

**Review by black23 — APPROVED** [review]

**Review by demiankatz — APPROVED** [review]
Thanks again, everyone!

**Pull request merged**
